pysat/_meta.py

```diff
@@ -188,7 +206,7 @@ def __init__(self, metadata=None,
         self._base_attr = dir(self)
 
     def __repr__(self):
-        """String describing MetaData instantiation parameters
+        """Print a representation of the Meta instantiation that can be called by `eval`.
```

Comment:
```suggestion
        """Print a representation of the Meta instantiation callable by `eval`.
```
This checks out locally for getting under 80 char